conf.py

```diff
@@ -54,4 +54,4 @@
     "dark_logo": "ArviZ_white.png",
 }
 
-intersphinx_mapping = {"arviz": ("https://arviz-devs.github.io/arviz/", None)}
+intersphinx_mapping = {"arviz": ("https://www.arviz.org/", None)}
```

Comment:
This should continue to be the python one, it is used to ease linking to the python docs and make the references more robust. 

We have for example a link to the contributing page in the python docs (and to the julia one) in https://github.com/arviz-devs/arviz-project/blob/main/our_team.md. The link to python one uses this intersphinx feature, now updating this, that and any other links to the python docs are updated simply by rebuilding the website.

```suggestion
intersphinx_mapping = {"arviz": ("https://python.arviz.org/en/latest/", None)}
```

Reply:
Aaah, okay, thanks for the explanation. My sphinx is rusty!

Reply:
It should work for non-sphinx generated docs as long as there is an `objects.inv` file with the right format indicating the labels and their respective url